doc/specs/stdlib_linalg.md

```diff
@@ -1565,4 +1565,53 @@ If `err` is not present, exceptions trigger an `error stop`.
 {!example/linalg/example_norm.f90!}
 ```
 
+## `mnorm` - Computes the matrix norm of a generic-rank array.
+
+### Status
+
+Experimental
+
+### Description
+
+This function computes one of several matrix norms of `real` or `complex` array \( A \), depending on 
+the value of the `order` input argument. \( A \) must be an array of rank 2 or higher. For arrays of rank > 2,
+matrix norms are computed over specified dimensions.
+
+### Syntax
+
+`x = ` [[stdlib_linalg(module):mnorm(interface)]] `(a, order [, dim, err])`
+
+### Arguments
+
+`a`: Shall be a rank-n `real` or `complex` array containing the data, where n � 2. It is an `intent(in)` argument.
+
+`order` (optional): Shall be an `integer` value or a `character` flag that specifies the norm type, as follows. It is an `intent(in)` argument. 
```

Comment:
Thanks for catching it @jvdp1. The argument is overall optional and I fixed the docs interface. To achieve that, because we have both `character` and `integer` versions, I only made it optional in the `integer` version, otherwise the interface would be flagged as ambiguous:

https://github.com/fortran-lang/stdlib/blob/706ea1c1b9719a34d83c1c2f7d3acf07083a210f/src/stdlib_linalg.fypp#L1382-L1383